db/python/layers/participant.py

```diff
@@ -210,13 +210,15 @@ async def fill_in_missing_participants(self):
             project=self.connection.project,
             allow_missing=True,
         )
-        if not unlinked_participants:
```

Comment:
Thanks for the fix! I was really unsure of what this was aiming to catch. Makes a lot of sense now.